taichi/ir/ir.cpp

```diff
@@ -92,14 +88,6 @@ IRNode *IRNode::get_ir_root() {
   return node;
 }
 
-Kernel *IRNode::get_kernel() const {
```

Comment:
These two methods are moved to `taichi/program/ir_node_extended_impl.cpp` because they need the definition of `Kernel` and `Program`.